app/lib/getNearbyServices.js

```diff
@@ -1,27 +1,49 @@
 const log = require('../../app/lib/logger');
-const docDB = require('../../config/config').docDB;
-const queries = require('../lib/queries');
-const filterServices = require('../lib/filterServices');
+const MongoClient = require('mongodb').MongoClient;
+const mongodbConfig = require('../../config/config').mongodb;
+const filterServices = require('./filterServices');
+const VError = require('verror').VError;
 
-function getNearbyServices(req, searchPoint, limits, next) {
-  const geoSpatialQuery = queries.geoSpatialQuery(searchPoint, limits.searchRadius);
+const connectionString = mongodbConfig.connectionString;
 
-  req.app.locals.docDBClient
-    .queryDocuments(docDB.collectionUrl, geoSpatialQuery)
-    .toArray((err, results) => {
-      if (err) {
-        log.error({ err }, 'DocumentDB query failed');
-        next(err);
-      } else {
-        log.debug(`${results.length} RESULTS returned...`);
+function getNearbyServices(searchPoint, limits, next) {
+  MongoClient.connect(connectionString).then((db) => {
+    const col = db.collection(mongodbConfig.collection);
 
-        log.debug('filter-and-sort-services-start');
-        const filteredServices = filterServices(results, limits);
-        log.debug('filter-and-sort-services-end');
+    col.aggregate([{
+      $geoNear: {
+        near: { type: 'Point', coordinates: searchPoint.coordinates },
+        distanceField: 'dist',
+        maxDistance: 32180,
+        distanceMultiplier: 0.001, // this is in miles
```

Comment:
The distance is in radians - 
`distanceMultiplier`	number	Optional. The factor to multiply all distances returned by the query. For example, use the distanceMultiplier to convert radians, as returned by a spherical query, to kilometers by multiplying by the radius of the Earth.

I was trying to understand how to convert it into miles: https://docs.mongodb.com/manual/tutorial/calculate-distances-using-spherical-geometry-with-2d-geospatial-indexes/#distance-multiplier